WordPress-Extra/ruleset.xml

```diff
@@ -103,6 +103,9 @@
 
 	<rule ref="WordPress.Security.EscapeOutput"/>
 
+	<!-- Encourage use of wp_safe_redirect() to avoid open redirect vulnerabilities. -->
```

Comment:
Please add the link to the issue being addressed by this sniff in the comment (as previously discussed and done elsewhere in the ruleset).